### PR TITLE
UI list Lambda Notification Targets

### PR DESCRIPTION
**Pull request by dvaldivia**
Screen Shot 2020-04-14 at 11 02 32 AM

### CODE REVIEW

**Comment by dvaldivia** [comment]
@bexsoft I purposely set it to grey xD "no news is good news"

File: portal-ui/src/screens/Console/NotificationEndopoints/ListNotificationEndpoints.tsx [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+                          return true;
+                        } else {
+                          return false;
+                        }
+                      }
+                    })
+                    .map(row => (
+                      <TableRow key={`${row.service}:${row.account_id}`}>
+                        <TableCell>{`${row.service}:${row.account_id}`}</TableCell>
+                        {/*<TableCell>{row.account_id}</TableCell>*/}
```
Comment (Alevsk) [inline portal-ui/src/screens/Console/NotificationEndopoints/ListNotificationEndpoints.tsx:177]: please delete this line

File: portal-ui/src/screens/Console/NotificationEndopoints/ListNotificationEndpoints.tsx [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+                            to={`/notification-endpoints/${row.service}:${row.account_id}`}
+                          >
+                            <IconButton aria-label="delete">
+                              <ViewIcon />
+                            </IconButton>
+                          </Link>
+                          <IconButton
+                            aria-label="delete"
+                            onClick={() => {
+                              //confirmDeleteBucket(row.name);
```
Comment (Alevsk) [inline portal-ui/src/screens/Console/NotificationEndopoints/ListNotificationEndpoints.tsx:206]: is this TODO ?

Comment (dvaldivia) [reply portal-ui/src/screens/Console/NotificationEndopoints/ListNotificationEndpoints.tsx:206]: yes, adding and deleting is work in progress, but I wanted to send at least the listing

**Comment by harshavardhana** [comment]
> @bexsoft I purposely set it to grey xD "no news is good news"

मौनम सम्मति लक्षणम (To be silent means I agree with you)

**Pull request merged**